<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pilototo_param_ui</class>
 <widget class="QDialog" name="pilototo_param_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instruction du pilototo</string>
  </property>
  <widget class="QGroupBox" name="position_grp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Position</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Longitude Est</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lon_deg">
       <property name="suffix">
        <string> °</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>-359</number>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lon_min">
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
       <property name="value">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Prochain cap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editWph"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Latitude Nord</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lat_deg">
       <property name="suffix">
        <string> °</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>-89</number>
       </property>
       <property name="maximum">
        <number>89</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lat_min">
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
       <property name="value">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_pastePOI">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>110</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Coller</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectPOI">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>110</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Selectionner un POI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copyPOI">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Copier</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>221</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>221</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>188</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mode de pilotage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cap / Angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="angle">
        <property name="minimum">
         <double>-179.990000000000009</double>
        </property>
        <property name="maximum">
         <double>179.990000000000009</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>mode</tabstop>
  <tabstop>angle</tabstop>
  <tabstop>lat_deg</tabstop>
  <tabstop>lat_min</tabstop>
  <tabstop>lon_deg</tabstop>
  <tabstop>lon_min</tabstop>
  <tabstop>editWph</tabstop>
  <tabstop>btn_pastePOI</tabstop>
  <tabstop>btn_selectPOI</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pilototo_param_ui</receiver>
   <slot>modeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pilototo_param_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>pilototo_param_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_pastePOI</sender>
   <signal>clicked()</signal>
   <receiver>pilototo_param_ui</receiver>
   <slot>pastePOI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_selectPOI</sender>
   <signal>clicked()</signal>
   <receiver>pilototo_param_ui</receiver>
   <slot>selectPOI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_copyPOI</sender>
   <signal>clicked()</signal>
   <receiver>pilototo_param_ui</receiver>
   <slot>copyPOI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modeChanged(int)</slot>
  <slot>pastePOI()</slot>
  <slot>selectPOI()</slot>
  <slot>copyPOI()</slot>
 </slots>
</ui>
